### ISSUE TITLE
Styling of availability in "Preferences"

### ISSUE THREAD

**Issue by ghost** [Jobcore-Talent/JC-Mobile-V.1#2]
# Styling of availability in "Preferences"

## Issue:
When you go to the "Preferences" page, there is a button that says "My Availability" and it right underneath it shows the days and times that an employee is available to work. Like this picture below:

![Image](https://user-images.githubusercontent.com/74715265/192567467-6710883e-b882-4113-a7d6-69c4d007edd9.jpg)

The issue is, that the dates and times could look better. For reference, see lines 358 to 396 from /workspace/employee-mobile/src/components/Invite/JobPreferences.js and line 108 of /workspace/employee-mobile/src/components/Invite/AvailabilityStyle.js

## Solution:
I provide a suggestion of how it could look in the picture below:

![Image](https://user-images.githubusercontent.com/74715265/192568178-2054108a-7ac9-44a1-ba8c-b8a4f169ef5d.jpg)

Basically, we can try to add line breaks to improve readability.

## How to solve it:
1. Try to see how it's being styled, and see where you could add line breaks.
2. Once finished, run all the tests provided in the repo to make sure that your integration was successful.
3. Create a pull request so that your code can be evaluated by the Lead Developer before deployment.
4. Keep an eye out for any new bugs that arise after the change is deployed to the code.
